src/components/user_view/UserInfo.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+
+export default class UserInfo extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  /***************************** Core functions *****************************/
+
+  /**************************** Visual component ****************************/
+
+  render() {
+    return (
+      <div>
+        <div>
+          <button className="btn btn-primary pull-right" type="submit">
```

Comment:
Since it isn't part of a form, having type="submit" is a bit weird. You could do type="button" which basically highlights it does nothing on its own and needs added functionality, or take you could take out type altogether. The default type is actually "submit" still, but at least it isn't highlighted as such...

Reply:
Ahh, right. I'll just take out the "type" field on this button.